doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -294,3 +294,29 @@ set with::
 :Type: Unsigned Integer
 :Required: No
 :Default: 64K
+
+SPDK Usage
+==================
+
+If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
+number here with "spdk:" prefix for ``bluestore_block_path``.
+Users can use 'lspci -vvv -d 8086:0953 | grep "Device Serial Number"' to
```

Comment:
For example, users can find the serial number with::

  $ lspci -vvv -d 808... | grep ...
  <show sampel output here>

and then set::

  bluestore block path = spdk:...